terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/FunctionTestSupport.java

```diff
@@ -187,15 +194,14 @@ protected void bootDefaultWebDriver() {
         if (driver == null) {
             driver = newWebDriver();
         }
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(
-                defaultTimeoutSecForImplicitlyWait));
+        driver.manage().timeouts().implicitlyWait(
+                defaultTimeoutSecForImplicitlyWait);
```

Comment:
- [ ] add `this.`

Reply:
177b3ef2380579720254cd32ccc95a34c5f9b896
added `this.`

Reply:
There are many more areas than we have corrected this time, but we will decide what to do about the other areas in the future in another place.